apps/examples/src/ExamplePage.tsx

```diff
@@ -86,8 +85,13 @@ export function ExamplePage({
 								<ul className="example__sidebar__category__items">
 									{examples
 										.find((category) => category.id === currentCategory)
-										?.value.filter((example) =>
-											example.title.toLowerCase().includes(filterValue.toLowerCase())
+										?.value.filter(
```

Comment:
this wasn't a problem you introduced but one thing we haven't gotten around to fixing here is splitting up the `filterValue` into separate terms and searching on each term.

so, if i filter for a "add tool" it won't find the "Add a tool to the Toolbar" example because i didn't include the "a".

if you want, it'd be great to do a search on each term separately!